From b246adf8af63cb908b17d74e8591e24798be6267 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Wed, 2 Nov 2011 14:10:19 +0800
Subject: [PATCH 0529/2508] ENGR00161207 - i.MX6/defconfig: add IEEE 1588
 config

If use IEEE 1588, need to enable below the config:

CONFIG_FEC_1588=y
CONFIG_OUT_OF_BAND=y

These config applied to i.MX5 and i.MX6Q.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 0228ca4..2755a44 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -884,6 +884,8 @@ CONFIG_SMSC911X=y
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_FEC=y
+# CONFIG_FEC_158 is not set
+# CONFIG_OUT_OF_BAND is not set
 # CONFIG_NETDEV_1000 is not set
 # CONFIG_NETDEV_10000 is not set
 CONFIG_WLAN=y
-- 
1.7.7.4

